### ISSUE TITLE
[Request] Attribute to exclude a class/method from advising.

### ISSUE THREAD

**Issue by ste-art** [ArxOne/MrAdvice#55]
When applying logger/debugger advice to an assembly you may want to exclude some classes/methods from logging (i.e some often executed methods to reduce log size or the logger class itself).
If auto-apply the same logic to advices it could also fix [Issue 49](https://github.com/ArxOne/MrAdvice/issues/49).

Something like:

``` C#
[assembly: MyAdvice]

    class Foo1
    {
        public void Bar1()
        {
            // advised
        }

        [IgnoreAdvice]
        public void Bar2()
        {
            // not advised
        }
    }

    [IgnoreAdvice]
    class Foo2
    {
        public void Bar1()
        {
            // not advised
        }
    }
```

**Comment by picrap** [comment]
That's an idea. What if we only want to avoid one advice (and apply others)?
Maybe we could extend the `IgnoreAdviceAttribute` with types or wildcards as parameters, such as:
```csharp
[IgnoreAdvice(typeof(SomeAdvice))]
class A {}

[IgnoreAdvice("*")]
class B {}

// and also mix them?
[IgnoreAdvice(typeof(SomeAdvice),"*LogAdvice")]
class C {}

```

**Comment by picrap** [comment]
Superseded by #75

**Comment by picrap** [comment]
Resolved in 2.3.0.
